charts/spire/charts/spire-server/values.yaml

```diff
@@ -370,6 +370,14 @@ notifier:
 controllerManager:
   ## @param controllerManager.enabled Flag to enable controller manager
   enabled: false
+
+  ## @param controllerManager.autoClassName if true, the controller class name will be Release.Namespace-Release.Name to automatically not conflict with other installs, allowing parallel installs.
+  autoClassName: true
+  ## @param controllerManager.className specify to use an explicit class name
+  className: ""
+  ## @param controllerManager.watchClassless specify to process custom resources without class name specified. Useful to migrate to class names from classless installs
```

Comment:
Can we follow the same pattern as the ingress, that we auto fill in the className if its empty string?

Reply:
Done